packages/browser-crypto/README.md

```diff
@@ -0,0 +1,33 @@
+![Coverage](https://img.shields.io/codecov/c/github/openwallet-foundation-labs/sd-jwt-js)
+![License](https://img.shields.io/github/license/openwallet-foundation-labs/sd-jwt-js.svg)
+![NPM](https://img.shields.io/npm/v/%40sd-jwt%2Fcore)
+![Release](https://img.shields.io/github/v/release/openwallet-foundation-labs/sd-jwt-js)
+![Stars](https://img.shields.io/github/stars/openwallet-foundation-labs/sd-jwt-js)
+
+# SD-JWT Implementation in JavaScript (TypeScript)
+
+## SD-JWT Browser Crypto
+
+### About
+
+Browser Crypto support for SD JWT
+
+Check the detail description in our github [repo](https://github.com/openwallet-foundation-labs/sd-jwt-js).
+
+### Installation
+
+To install this project, run the following command:
+
+```bash
+npm install @sd-jwt/crypto-browser
```

Comment:
should we mention in all the readme that they also could use yarn or pnpm?

Reply:
I'll add yarn and pnpm :)